package.json

```diff
@@ -9,10 +9,10 @@
     "lint": "next lint",
     "test": "jest tests/unit",
     "lint:fix": "eslint --fix */**/*.js tests/**/*.js",
-    "cy:open": "cypress open --env CYPRESS_RELEASE_VERSION=full",
-    "cy:openmvp": "cypress open --env CYPRESS_RELEASE_VERSION=mvp",
-    "cy:run": "cypress run --env CYPRESS_RELEASE_VERSION=full",
-    "cy:runmvp": "cypress run --env CYPRESS_RELEASE_VERSION=mvp  --spec 'tests/cypress/integration/reportRepair/*Mvp.spec.js'",
+    "cy:open": "cypress open --env RELEASE_VERSION=full",
+    "cy:openmvp": "cypress open --env RELEASE_VERSION=mvp",
```

Comment:
Shall we unify `:mvp` vs `mvp` etc?

Reply:
This was done in this [commit](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/commit/3157b69e0520a4aae608668b4968003b40294356).